### DISCUSSION TITLE
User/Customer portal email notification expire

### DISCUSSION THREAD

**Discussion by flopp999** [keygen-sh/community#81] [Q&A] [answered]
I tested to create a user.
Is there any portal for the user?
After a reset password the user came to admin login page.

What I want to find is if the user can get an email when the license has xx days left before expires.

**Comment by ezekg** [comment]
Currently, we don't offer a customer-facing portal. But we're actively working on offering this very soon. For email notifications, you'd need to listen for the `license.expiring-soon` webhook, or have a periodic job that lists licenses expiring within the last *n* days (e.g. using the `?expires[in]=30d` filter), and then send the emails yourself. You can also do the former [with Zapier](https://zapier.com/apps/keygen/integrations), if you want a no-code solution.

Other than password reset emails, we don't send user-facing emails.

**Comment by flopp999** [comment, marked as answer]
Thanks, I will go for a cronjob and API
